### DISCUSSION TITLE
[Question]: Should `KToast` be moved to EXT repo until stable ?

### DISCUSSION THREAD

**Discussion by Smurf-IV** [Krypton-Suite/Standard-Toolkit#1616] [General]
e.g. #1287 
#1282

**Comment by PWagner1** [comment]
Hi @Smurf-IV 

I don't think that it should be pulled from the ST in it's current state, as it would cause chaos for devs who might be using it. **But** I'm currently working on an improved version in the ET, which will be stable enough to replace the current iteration in V100.
